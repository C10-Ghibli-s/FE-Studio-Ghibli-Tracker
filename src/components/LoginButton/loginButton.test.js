import React from 'react';
import TestRenderer from 'react-test-renderer';
import { LoginButton } from '.';

const testRenderer = TestRenderer.create(
  <LoginButton page="/login">Login</LoginButton>
).toJSON();


// test('match spanshot link', () => {
//     expect(testRenderer.props.href).toBe('https://www.facebook.com/')
// })
test('When the button is clicked needs to redirect to the Login page', () => {
    expect(testRenderer).toMatchSnapshot();
})

// test('match snapshot with route :v', () => {
//     expect(testRenderer.props.href).toBe('/');
// })