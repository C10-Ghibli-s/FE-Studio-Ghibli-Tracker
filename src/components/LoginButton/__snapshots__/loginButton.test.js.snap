// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`When the button is clicked needs to redirect to the Login page 1`] = `
<a
  className="loginButton"
  href="/login"
>
  Login
</a>
`;
